input_image: "/usb_cam/image_raw"
output_image: "/mobilenet/image_raw"
prediction_topic: "/mobilenet/prediction"